File: MPI/mpicheck/tests/functions/3.0/MPI_Mprobe.f

        
        program main
        include 'mpif.h'
        
        INTEGER var_0
       INTEGER var_1
       INTEGER var_2
       INTEGER var_3
       INTEGER var_4(10)
       INTEGER var_5
        call mpi_mprobe(var_0, var_1, var_2, var_3, var_4, var_5)
       call pmpi_mprobe(var_0, var_1, var_2, var_3, var_4, var_5)
        end program main
    

File: MPI/mpicheck/tests/functions/3.0/MPI_Mprobe.c
#include <mpi.h>
int main(char argc, char**argv)
{
    /* vars */
    int var_0;
    int var_1;
    MPI_Comm var_2;
    MPI_Message *var_3;
    MPI_Status *var_4;
    int ret;
    /* calls */
    ret = MPI_Mprobe(var_0, var_1, var_2, var_3, var_4);
    ret = PMPI_Mprobe(var_0, var_1, var_2, var_3, var_4);
    return 0;
}
